maintainers/maintainer-list.nix

```
@@ -14403,6 +14403,12 @@
     githubId = 2287221;
     name = "Andreas Zweili";
   };
+  nebunebu = {
+    email = "neb.nebuchadnezzar@gmail.com";
+    github = "nebunebu";
+    githubId = "87451010";
```

`githubId` should be numeric:
```suggestion
    githubId = 87451010;
```